test/controllers/api/v1/schools_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Api::V1::SchoolsControllerTest < ActionDispatch::IntegrationTest
+  test "test_schools_are_created" do
+    assert_equal 2, School.count
+  end
+end
```

Comment:
Use rspec for tests, because that's just how we roll.  https://relishapp.com/rspec/rspec-rails/docs/gettingstarted

Reply:
updated to use rspec